packages/core/admin/admin/src/components/LeftMenu.tsx

```diff
@@ -134,19 +141,25 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }: LeftMenuProps) =
             })}
           >
             {pluginsSectionLinks.map((link) => {
-              const Icon = link.icon;
-
+              const LinkIcon = link.icon;
               return (
-                <NavLink
-                  as={RouterNavLink}
-                  // @ts-expect-error the props from the passed as prop are not inferred // joined together
-                  to={link.to}
-                  key={link.to}
-                  icon={<Icon />}
-                  onClick={() => handleClickOnLink(link.to)}
-                >
-                  {formatMessage(link.intlLabel)}
-                </NavLink>
+                <NavLinkWrapper key={link.to}>
+                  <NavLink
+                    as={RouterNavLink}
+                    to={link.to}
+                    icon={<LinkIcon />}
+                    onClick={() => handleClickOnLink(link.to)}
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore: we need to change the badgeContent in the DS to accept not just only strings but also other components and also configure the badge background color
```

Comment:
```suggestion
                    // @ts-expect-error: we need to change the badgeContent in the DS to accept not just only strings but also other components and also configure the badge background color
```
Your comment is quite long for a single line might read better on multiple lines but up to you the linter doesn't seem to complain